admin/views/tool-file-editor.php

```diff
@@ -30,7 +30,7 @@
 
 if ( isset( $_POST['submitrobots'] ) ) {
 	if ( ! current_user_can( 'edit_files' ) ) {
-		die( __( 'You cannot edit the robots.txt file.', 'wordpress-seo' ) );
+		die( esc_html__( 'You cannot edit the robots.txt file.', 'wordpress-seo' ) );
```

Comment:
I know this is not the scope of this pr, but I think we should get `robots.txt` out of this string

Reply:
Done.